sprout/common/__init__.py

```diff
@@ -1,35 +1,35 @@
-from flask import Flask
-from flask_sqlalchemy import SQLAlchemy
-from flask_marshmallow import Marshmallow
-from flask_migrate import Migrate
-from sprout.config import Config
-from sprout.extensions import db, ma, make_celery
-from sprout.routes import ai_bp, job_bp
+# from flask import Flask
+# from flask_sqlalchemy import SQLAlchemy
+# from flask_marshmallow import Marshmallow
+# from flask_migrate import Migrate
+# from sprout.config import Config
+# from sprout.extensions import db, ma, make_celery
+# from sprout.routes import ai_bp, job_bp
```

Comment:
Why do we comment these out ?

Reply:
This was copied from the root directory, and keeping it may cause compilation errors.